docs/clusterjoinprocess.md

```diff
@@ -56,6 +54,6 @@ based on the following steps:
 - check if organization field and commonName field is valid.
 - check if user name in csr is the same as commonName in certificate to ensure the request
 is originated from the same identity.
-- check if the corresponding `ManagedCluster` is in the conndition of `HubApprovedJoin`.
+- check if the corresponding `ManagedCluster` is in the conndition of `HubAcceptedManagedCluster`.
```

Comment:
I did not find this check logic in [hub-csr-controller](https://github.com/open-cluster-management-io/registration/blob/main/pkg/hub/csr/controller.go), should we remove this, or add the check logic into hub-csr-controller?
cc @qiujian16 @skeeey

Reply:
oh let's remove this, accepted is only about authz. So we do not need to check this for csr.